src/webui/modules/home/index.js

```diff
@@ -98,12 +105,10 @@ export default class Home extends React.Component {
   }
 
   render() {
-    return (
-      <div>
+    return <div>
         {this.renderSearchBar()}
-        {this.state.loading ? this.renderLoading() : this.renderPackageList()}
-      </div>
-    );
+      {this.state.loading ? this.renderLoading() : <PackageList help={this.state.packages.length === 0} packages={this.state.packages} />}
```

Comment:
`this.state.packages.length`
lodash provides here `isEmtpy` which is kinda safe

Reply:
you can destructure state here as well.